### PR TITLE
ENCD-4089-Deactivated-rep-num-in-annot-table

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

File: src/encoded/static/components/dataset.js [hunk 1]
```diff
@@ -225,7 +225,7 @@ class AnnotationComponent extends React.Component {
                 </Panel>
 
                 {/* Display the file widget with the facet, graph, and tables */}
-                <FileGallery context={context} encodevers={encodevers} />
+                <FileGallery context={context} encodevers={encodevers} showReplicateNumber={false}  />
```
Comment (caseylitton) [inline src/encoded/static/components/dataset.js]: I think there are two spaces after 
```
showReplicateNumber={false}
```
The ticket is already merge ready, but I guess we should fix it?

Comment (forresttanaka) [reply src/encoded/static/components/dataset.js]: Probably should be fixed. Looks like it causes an eslint error. @zoldello Do you have eslint configured in Atom in a way that uses the encoded .eslintrc file?

Comment (caseylitton) [reply src/encoded/static/components/dataset.js]: Sounds good.  I'll mark this in PR as review requested.  I don't think we need a new QA though, right?

File: src/encoded/static/components/dataset.js [hunk 2]
```diff
@@ -225,7 +225,7 @@ class AnnotationComponent extends React.Component {
                 </Panel>
 
                 {/* Display the file widget with the facet, graph, and tables */}
```
Comment (caseylitton) [inline src/encoded/static/components/dataset.js]: Please fix eslint error with the double spaces in the other comment.  Check with Forrest, but we probably don't need a new demo/qa cycle.

File: src/encoded/static/components/dataset.js (hunk 1 above)
Comment (caseylitton) [reply src/encoded/static/components/dataset.js]: oh, it showed up in codacy.  I should look at that more

**Pull request merged**